demo-cat-gallery/build.gradle

```diff
@@ -36,11 +37,15 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    sourceSets {
+        main.java.srcDirs = ['src/main/kotlin']
```

Comment:
There will be only kotlin files under the srcDires. So let's remove this and use the default src/main/java srcDirs and move the kotlin files under  src/main/java